pageserver/src/tenant/storage_layer/inmemory_layer.rs

```diff
@@ -339,4 +306,22 @@ impl InMemoryLayer {
         let delta_layer = delta_layer_writer.finish(Key::MAX)?;
         Ok(delta_layer)
     }
+
+    pub fn get_current_lsn_range(&self) -> Range<Lsn> {
+        let inner = self.inner.read().unwrap();
+
+        let end_lsn = if let Some(end_lsn) = inner.end_lsn {
+            end_lsn
+        } else {
+            Lsn(u64::MAX)
+        };
```

Comment:
```suggestion
        let end_lsn = inner.end_lsn.unwrap_or(Lsn(u64::MAX));
```

To align with the new line 324.